backend/src/exercise/http-server.ts

```diff
@@ -14,11 +14,19 @@ import { secureHttp } from './http-handler/secure-http';
 
 export class ExerciseHttpServer {
     public readonly httpServer: HttpServer;
-    constructor(app: Express, port: number, databaseService: DatabaseService) {
+    /**
+     * @param uploadLimit in Megabyte can be set via ENV DFM_UPLOAD_LIMIT
+     */
+    constructor(
+        app: Express,
+        port: number,
+        uploadLimit: number,
+        databaseService: DatabaseService
+    ) {
         // TODO: Temporary allow all
         app.use(cors());
 
-        app.use(express.json({ limit: '200mb' }));
+        app.use(express.json({ limit: String(`${uploadLimit}mb`) }));
```

Comment:
Two things here:

1. Are you sure you need to call `String()` here? Wouldn't it be enough to just use the format string?
2. I think it would be better to directly access the config here instead of passing the limit as a parameter. I'm not sure though.

Reply:
String( ) is probably not needed.

For the orher, then also the port should be reqd directly from config or not?

Reply:
I agree with 2, so only remove the `String()` call and this is good to go.

Reply:
@ClFeSc removing the port (httpPort with or without webSocketPort) seems to break cypress, I am not sure why, it should work, maybe something about initialization missing in cypress test?

Reply:
I don't know what you mean. You can't just remove parts of the program and expect it to work like before.